### PR TITLE
pkp/pkp-lib#8734 Removed extra process call to handle bulk email sending

### PR DESCRIPTION
**Pull request by touhidurabir**
PR for pkp/pkp-lib#8734

### CODE REVIEW

File: src/components/Container/AccessPage.vue [hunk 1]
```diff
@@ -1,48 +1,19 @@
...
 		NotifyUsersForm,
-		ProgressBar,
 	},
 	data() {
 		return {
-			completedJobs: 0,
-			progressUrl: '',
-			queueId: '',
-			totalJobs: 0,
+			totalBulkJobs: 0,
```
Comment (Vitaliy-1) [inline src/components/Container/AccessPage.vue:13]: Is this data used anywhere? I think a total number of jobs was here to inform about the progress of processing jobs.

Comment (touhidurabir) [reply src/components/Container/AccessPage.vue:13]: Thats has used there as an reference to confirm that all jobs has been pushed to queue successfully and display following view to confirm users 

CleanShot 2023-03-25 at 16 57 53@2x

we can also just pass a simple boolean flag to to confirm and present the view also and I think that better as passing number of batches make no sense there .

Comment (touhidurabir) [reply src/components/Container/AccessPage.vue:13]: Thinking about it again , passing a boolean flag or number to successfully dispatched batches don't make much difference just to show the success view through vue

Comment (Vitaliy-1) [reply src/components/Container/AccessPage.vue:13]: Yes, you are right, I see it now. The concept on how the queue is processed has changed here and the form likely should be reconsidered but for now it looks good. Thanks!

**Pull request merged**